### DISCUSSION TITLE
Windows no longer recognizes MicroSD card

### DISCUSSION THREAD

**Discussion by lokisilverblade** [OnionUI/Onion#1196] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

Onion V4.2.0-RC

### Miyoo Firmware Version

latest

### Provide a clear and concise description of the issue

Hello, I have a problem and Windows no longer recognizes the Micro SD card. I have already tested it with two different computers and card readers and under Linux but always the same result. I can't open my card on the PC. Since I would like to save my saves, I ask for urgent help because I have already invested a few hours of play and I think it would be a shame if I had to throw it away now. Thanks.

### Steps to Reproduce

No way to test how i can reproduce it.

### Provide any additional context or information that may help us investigate your issue

Everything is above

**Comment by thomasklemenc** [comment]
Sorry to say but since you've already tried multiple SD card readers and operating systems it looks like your card is dead, not much you can do about it sans paying for data recovery.

**Reply by lokisilverblade** [reply]
Hey, the problem is the sd card boots into onion os without problems and all games working. So I don't think the SD card is defective.

**Reply by thomasklemenc** [reply]
Well that changes things. Are you sure it boots into Onion and not the stock OS which it boots into without an SD card?
If it boots into Onion and you have the Miyoo Mini Plus you could use network features to transfer the save games.

**Reply by XK9274** [reply]
> Hey, the problem is the sd card boots into onion os without problems and all games working. So I don't think the SD card is defective.


I have this exact same problem on my main MMP. 

It doesn't change much, backup your saves etc with a network feature like Gecko says then reformat the SD, if you choose to continue using the SD in its current state it's highly recommended you setup some sort of cloud/save sync of critical directories or regularly back them up as it's probably due a reformat in any case.

**Reply by lokisilverblade** [reply]
Yes im sure that onion os boots Up. How can i Backup my files with wifi? Thanks.

**Comment by Aemiii91** [comment]
Which model do you have? Mini or Plus?

**Reply by lokisilverblade** [reply]
Miyoo Mini Plus

**Comment by ropeFullOfHope** [comment]
This has happened to me too. Twice at that. I'm using a Sandisk 128gb uSD card.
The first time it was on v4.3.0-RC while I tried launching SM64DS. I got a black screen and when I went to check my uSD card it only contained one extensionless file with random symbols for name. My MM+ would still be able to read ROMs and other files, but the file browser would immediately close if I tried launching it.
The second time happend less than a week later, where I simply tried to resume Gran Turismo 2 that I played a few hours before. I launched my MM+ and got booted into stock firmware. I checked my uSD card and it contained only a file named cachefile. I used H2testw to test my uSD card and got no errors.
Browsing on the miyoo subreddit I saw that I'm not the only one. I could try buying a new uSD card, but I need proof that something is wrong with my current one.
Any help?

**Comment by Phanturian** [comment]
I think this might be an issue with the OTA updater that was introduced sometime after 4.2.0. Back in November 2023, users were reporting issues with their SD cards on Mac after using the OTA updater as you can [see here](https://github.com/OnionUI/Onion/discussions/1249). Something in the OTA process was causing the SD card to become unrecognizable or appear to be empty on a computer, but when the card is put into the Mini Plus, all the games are there and playable. They had to run first aid on the partition to make the files appear again. This is the first I've heard of a similar issue with PC users. I have not been using the OTA updater since these issues were first reported. If somebody loads up their SD card, uses the OTA updater, but doesn't go back to add more games, they may never know the issue exists.

**Reply by ropeFullOfHope** [reply]
OTA updates could be their own seperate issue or even a non-issue, as I got my uSD card corrupted twice without ever using OTA updates.

**Comment by AdvanDizdarevic** [comment]
Try this https://www.youtube.com/watch?v=J5xFqEUuwYs, I was stupid to rename my card without assigning my card a partition letter.

EDIT: In case the video disappears or people don't feel like watching the steps are
- open your windows search and type: "create and format hard disk partitions"
- in the list of devices you should see your SD card without an assigned letter
- right click and select "Change drive letter and Paths"
- if there's nothing in the list just click add and assign it a letter
- once you click ok it should immediately recognize your SD card.

Hope it helps :)

**Reply by PedroGuerraPT** [reply]
It did help, thank you!
